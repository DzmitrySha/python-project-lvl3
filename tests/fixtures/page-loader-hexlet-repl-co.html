<!DOCTYPE html>
<html lang="ru">
    <head>
        <meta charset="utf-8">
        <title>Курсы по программированию Хекслет</title>
        <link rel="stylesheet" media="all" href="https://cdn2.site.com/blog/assets/style.css">
        <link rel="stylesheet" media="all" href="/assets/application.css" />
        <link href="/courses" rel="canonical">
    </head>
    <body>
        <img src="/assets/professions/nodejs.png" alt="Иконка профессии Node.js-программист" />
        <h3>
            <a href="/professions/nodejs">Node.js-программист</a>
        </h3>
    </body>
    <script src="https://js.stripe.com/v3/"></script>
    <script src="/script.js"></script>
</html>
