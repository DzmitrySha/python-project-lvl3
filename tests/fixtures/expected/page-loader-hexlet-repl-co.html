<!DOCTYPE html>
<html lang="ru">
    <head>
        <meta charset="utf-8">
        <title>Курсы по программированию Хекслет</title>
        <link rel="stylesheet" media="all" href="https://cdn2.page-loader.hexlet.repl.co/blog/assets/style.css">
        <link rel="stylesheet" media="all" href="page-loader-hexlet-repl-co_files/page-loader-hexlet-repl-co-assets-application.css" />
        <link href="page-loader-hexlet-repl-co_files/page-loader-hexlet-repl-co-courses.html" rel="canonical">
    </head>
    <body>
        <img src="page-loader-hexlet-repl-co_files/page-loader-hexlet-repl-co-assets-professions-nodejs.png" alt="Иконка профессии Node.js-программист" />
        <h3>
            <a href="/professions/nodejs">Node.js-программист</a>
        </h3>
    </body>
    <script src="https://js.stripe.com/v3/"></script>
    <script src="page-loader-hexlet-repl-co_files/page-loader-hexlet-repl-co-script.js"></script>
</html>
